### DISCUSSION TITLE
Change the old looking dropdown menu!

### DISCUSSION THREAD

**Discussion by Akascape** [TomSchimansky/CustomTkinter#1577] [Showcase]
**Here is an easy to use scrollable dropdown menu!**
## [CTkScrollableDropdown](https://github.com/Akascape/CTkScrollableDropdown)

![screenshots](https://user-images.githubusercontent.com/89206401/236677843-8d8b76fd-6145-47b1-8f4d-b6a64b08e1ea.png)

## Features
- Rounded corners
- **Define custom height for the menu**
- Automatic resize
- Transparency effects
- **Live search options**
- Full customisability
- Add images to options
- Automatic bindings added for ctkoptionmenu/ctkcombobox

**Comment by Quantlyx** [comment]
Excited to use this in a project of mine.
However I'm trying to implememt the scrollable frame of the dropdown as a regular widget into my app.
So far not able to accomplish that due to my limited knowledge of writing custom widgets.
Any suggestions about how I go at it or some resources to get me started?

**Reply by Akascape** [reply]
@Quantlyx 
Analyse the code of ctkscrollabledropdown and add your widgets in the that frame, remove the default buttons and bindings and add your own widgets/functions there.
